CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required( VERSION 2.8 FATAL_ERROR )
 project (urdfdom_headers)
 
+include(GNUInstallDirs)
```

Comment:
hmm, nvm. it looks like this same include is used in `urdfdom` already. So it's probably ok.